### DISCUSSION TITLE
Possible to limit run to manual actions?

### DISCUSSION THREAD

**Discussion by fgblomqvist** [renovatebot/renovate#10131] [Request Help] [answered]
To reduce noise, we only run Renovate once a day, very early in the morning. Some schedules also limit things further by only specifying certain days of the week or month.

However, this greatly limits velocity in terms of dealing with things like merge conflicts. PRs don't get rebased until the next morning and manual actions, like marking a PR to be rebased, don't run until then either.

Is it possible to run Renovate and only have it deal with manually specified actions? E.g. any boxes checked anywhere. Would make it possible to run it every 30 minutes or so without generating unnecessary noise. Or are we going about this the wrong way?

**Comment by rarkins** [comment, marked as answer]
Our approach for this is not fully baked yet but it is:
- Use `schedule` to control when branches are created
- Allow configuration of *what* should be done outside schedules: #4866

This way you could keep Renovate running regularly but lock down what you allow it to do (the list in #4866 may need expanding by now.. it's a little old)

**Reply by fgblomqvist** [reply]
Yeah #4866 would indeed be necessary, but yeah okay, running once a night will have to suffice for now then.
